src/CiviliansAccountsMicroservice/Civilians.Infrastructure/DbContext/Configuration/RefreshTokensConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Civilians.Core.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace Civilians.Infrastructure.DbContext.Configuration
+{
+    internal static class RefreshTokensConfiguration
+    {
+        internal static void ConfigureRefreshTokensTable(this ModelBuilder builder)
```

Comment:
You should iheret from IEntityTypeConfiguration. Then you can apply them all in onmodelcreating just in one line